src/components/character-controller.js

```diff
@@ -0,0 +1,300 @@
+import AFRAME from "aframe";
+var CLAMP_VELOCITY = 0.01;
+var MAX_DELTA = 0.2;
+var TAU = Math.PI * 2;
+
+// Does not have any type of collisions yet.
+AFRAME.registerComponent("character-controller", {
+  schema: {
+    groundAcc: { default: 10 },
+    easing: { default: 8 },
+    pivot: { type: "selector" },
+    snapRotationRadian: { default: TAU / 8 }
```

Comment:
Maybe just use `45 * THREE.DEG2RAD` here and get rid of `TAU`.

Reply:
Sure, will use THREE.Math.DEG2RAD instead.